### PR TITLE
add quartz dependency

### PR DESCRIPTION
**Pull request by IvanLinnik**
add scheduler folder with quartz.properties, scheduler and job classes

it's just for testing for now. tell me if i'm on the wrong way.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -25,6 +25,9 @@ dependencies {
     annotationProcessor 'org.springframework.boot:spring-boot-configuration-processor'
     annotationProcessor 'org.projectlombok:lombok'
     testImplementation 'org.springframework.boot:spring-boot-starter-test'
+
+    // https://mvnrepository.com/artifact/org.quartz-scheduler/quartz
+    implementation group: 'org.quartz-scheduler', name: 'quartz', version: '2.3.2'
```
Comment (mmarusich) [inline build.gradle]: use starter from spring boot

File: src/main/java/io/skai/okta/internshipstreamingserviceparserxml/scheduler/CheckUpdateScheduler.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import org.quartz.impl.StdSchedulerFactory;
+
+import static org.quartz.JobBuilder.newJob;
+
+@Slf4j
+public class CheckUpdateScheduler {
+    public static void main(String[] args) {
+        try {
+            Scheduler scheduler = StdSchedulerFactory.getDefaultScheduler();
+            scheduler.start();
```
Comment (mmarusich) [inline src/main/java/io/skai/okta/internshipstreamingserviceparserxml/scheduler/CheckUpdateScheduler.java]: if you will use starter, spring boot auto configuration will do this job

File: src/main/java/io/skai/okta/internshipstreamingserviceparserxml/scheduler/quartz.properties [hunk 3]
```diff
@@ -0,0 +1,3 @@
+org.quartz.scheduler.instanceName = StreamingServiceScheduler
```
Comment (mmarusich) [inline src/main/java/io/skai/okta/internshipstreamingserviceparserxml/scheduler/quartz.properties]: don't create separate properties file, place this settings into regular application.properties file

File: src/main/java/io/skai/okta/internshipstreamingserviceparserxml/scheduler/SpringQuartzConfig.java [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+import static org.quartz.SimpleScheduleBuilder.simpleSchedule;
+
+@Configuration
+public class SpringQuartzConfig {
+    @Bean
+    public JobDetail jobDetail() {
+        return JobBuilder.newJob().ofType(CheckUpdateJob.class)
+                         .storeDurably()
+                         .withIdentity("Updater")
```
Comment (mmarusich) [inline src/main/java/io/skai/okta/internshipstreamingserviceparserxml/scheduler/SpringQuartzConfig.java]: use job key for identity,

example https://github.com/kenshoo/fb-product-fetcher/blob/master/application/src/main/java/io/skai/fbproductfetcher/quartz/config/QuartzJobConfig.java#L19

File: src/main/java/io/skai/okta/internshipstreamingserviceparserxml/scheduler/SpringQuartzConfig.java [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+                         .withDescription("Invoke Lostfilm RSS update service...")
+                         .build();
+    }
+
+    @Bean
+    public Trigger trigger(JobDetail job) {
+        return TriggerBuilder.newTrigger().forJob(job)
+                             .withIdentity("Time_Trigger")
```
Comment (mmarusich) [inline src/main/java/io/skai/okta/internshipstreamingserviceparserxml/scheduler/SpringQuartzConfig.java]: same

Comment (IvanLinnik) [reply src/main/java/io/skai/okta/internshipstreamingserviceparserxml/scheduler/SpringQuartzConfig.java]: done

File: src/main/java/io/skai/okta/internshipstreamingserviceparserxml/scheduler/SpringQuartzConfig.java (hunk 4 above)
Comment (IvanLinnik) [reply src/main/java/io/skai/okta/internshipstreamingserviceparserxml/scheduler/SpringQuartzConfig.java]: done

**Review by mmarusich — APPROVED** [review]

**Pull request merged**